test/packages/other/hit_count_assertion/manifest.yml

```
@@ -0,0 +1,24 @@
+format_version: 1.0.0
```

Nit. Use a more recent version of the spec.
```suggestion
format_version: 2.5.0
```